utils/UUID.hh

```diff
@@ -228,4 +226,8 @@ struct hash<utils::UUID> {
         return size_t((hilo >> 32) ^ hilo);
     }
 };
+
+inline std::ostream& operator<<(std::ostream& out, const utils::UUID& uuid) {
+    return out << uuid.to_sstring();
 }
```

Comment:
Why make it inline?

Reply:
it has to be inline when moved to the header file.
but actually this was done in an earlier version that tried to base utils::UUID itself and the base template, but I think this is no longer needed.

Reply:
Why can't you have a declaration but not a definition?

Reply:
As I tried to explain above, I can.
It was a preparation patch from an older version that is no longer needed.